.github/ISSUE_TEMPLATE/add-dataset.yml

```diff
@@ -0,0 +1,37 @@
+name: Add dataset
+description: Request the addition of a specific dataset to the Hub
+labels: ["dataset request"]
+body:
+  - type: input
+    id: name
+    attributes:
+      label: Name
+      description: Name of the dataset
+    validations:
+      required: true
+  
+  - type: input
+    id: paper
+    attributes:
+      label: Paper
+      description: Link to the dataset paper if available
+
+  - type: input
+    id: data
+    attributes:
+      label: Data
+      description: Link to the Github repository or current dataset location
+    validations:
+      required: true
+  
+  - type: textarea
+    id: motivation
+    attributes:
+      label: Motivation
+      description: What are some good reasons to have this dataset
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: Instructions to add a new dataset can be found [here](https://github.com/huggingface/datasets/blob/main/ADD_NEW_DATASET.md).
```

Comment:
Or maybe replace it with https://huggingface.co/docs/datasets/share

Reply:
Good idea! This template allows less experienced users to request the addition of a specific dataset to the Hub, which means they probably don't know how to share it themselves, so I think it's better to remove this section instead of replacing it with the Share doc page.

Reply:
But the issue could be self-assigned by other user, who might find the link useful.

Anyway, as you prefer, not a string opinion on this...

Reply:
Okay, I reverted the change and updated the link.